src/app/shell/admin-tools/data/provider-list/provider-list.component.html

```diff
@@ -2,26 +2,19 @@
   <div class="search-area">
     <div class="search-bar" fxLayout="row" fxLayoutAlign="flex-start center">
       <mat-icon class="search-icon" style="transform: rotate(90deg)">search</mat-icon>
-      <input
-        formControlName="searchBarFilter"
-        matInput
-        type="text"
-        class="search"
+      <input formControlName="searchBarFilter" matInput type="text" class="search"
         placeholder="{{ 'FORMS.PLACEHOLDERS.SEARCH_PROVIDER_EDRPO_MAIL_PHONE' | translate }}" />
     </div>
     <div class="reset" (click)="onResetFilters()" *ngIf="!isAreaAdmin">
-      <mat-icon class="reset__icon">refresh</mat-icon><span class="reset__title">{{ 'HISTORY_LOG.RESET_FILTERS' | translate }}</span>
+      <mat-icon class="reset__icon">refresh</mat-icon><span class="reset__title">{{ 'HISTORY_LOG.RESET_FILTERS' |
```

Comment:
Same here, this formatting looks odd.
We can do something like this:
```
<mat-icon class="reset__icon">refresh</mat-icon>
<span class="reset__title">{{ 'HISTORY_LOG.RESET_FILTERS' | translate }}</span>
```

Reply:
Honestly, it's the autoformatting's actions, so will pay more attention on such changes